app/components/blacklight/system/dropdown_button_component.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Blacklight
+  module System
+    class DropdownButtonComponent < Blacklight::Component
+      def initialize(label:, classes: %w[btn btn-outline-primary dropdown-toggle])
+        @classes = classes
```

Comment:
change is here ... it uses `btn-outline-primary` instead of `btn-outline-secondary`

Reply:
Can we just extend Blacklight component here and change the default param, and call `super`?

Reply:
I see that you've completely replaced the BL module.  I think overriding it in our root namespace might be more understandable.

Reply:
Can you give me an example of how you imagine this working? This was the approach I started with, but I found I had to override/sublcass two other BL components and a view in order to get to to render the `Earthworks` version of the components, and it felt a bit convoluted for a single class change.  Maybe I'm missing something.

Reply:
I'm thinking https://github.com/projectblacklight/blacklight/pull/3289 might help.  Once that's in. I can look at https://github.com/projectblacklight/blacklight/issues/3290 which should make this trivial.

Reply:
So, https://github.com/projectblacklight/blacklight/pull/3296 now will allow this component to be configured.

Reply:
Thanks Justin.  For this PR, I just need to override the class for the button that is rendered, which is in`DropdownButtonComponent`: https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/system/dropdown_button_component.rb  

It looks like the reference to the button component is still hardcoded within the `DropdownComponent`: https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/system/dropdown_component.rb#L6

If I create new a Earthworks `DropDownComponent`, subclass it from the Blacklight component, use the new blacklight config to tell it to use this new Earthworks::DropdownComponent, then create a new Earthworks specific `DropdownButtonCompontent` (also subclassed from the blacklight one) so that I can tell it render with my new css class (e.g. just do this in my subclass `DropDownComponent`: `renders_one :button, Earthworks::System::DropdownButtonComponent`), it complains of a duplicate slot:

```
Earthworks::System::DropdownComponent declares the button slot multiple times.
To fix this issue, choose a different slot name.
```

Reply:
Maybe if this component https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/system/dropdown_component.rb#L6 was configurable in blacklight in a similar way to what you did in https://github.com/projectblacklight/blacklight/pull/3296, I could create a new Earthworks `DropdownButtonComponent`, subclass from the blacklight one and update the css class as needed.

Reply:
I'm not sure I understood, but presently, BL allows you to configure which DropdownComponent you want to use.